lib/usbmux/transformer/usbmux-decoder.js

```diff
@@ -0,0 +1,32 @@
+import { parse } from '../../util/plist';
+import Stream from 'stream';
+
+const HEADER_LENGTH = 16;
+
+class UsbmuxDecoder extends Stream.Transform {
+
+  constructor () {
+    super({ objectMode: true });
+  }
+
+  _transform (data, encoding, callback) {
+    this._decode(data);
+    callback();
+  }
+
+  _decode (data) {
```

Comment:
it makes sense to add docstrings to methods, since the argument types are not obvious

Reply:
Is it? It is just an internally called function. In theory, nothing else should call this function from outside